src/content/guides/how-to-use-a-wallet/index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: How to use a wallet
+description: A guide explaining how to send, receive tokens and connect to web3 projects.
+lang: en
+---
+
+# How to: Use a wallet
+
+Learn how to operate all the basic functions of a wallet. If you don’t have one yet, check out our [How to: "Register" an Ethereum account](/guides/how-to-register-an-ethereum-account/).
+
+## Open your wallet
+
+You should see a dashboard that will likely show your balance and contain buttons to send and receive tokens. 
+
+## Receive cryptocurrency
+
+Do you want to receive crypto into your wallet?
+
+Each Ethereum account has its own receiving address which is a unique sequence of numbers and letters. The address functions like a bank account number. Ethereum addresses will always start with “0x”.
+
+You need to provide the sender with your receiving address. Many wallet apps let you copy to clipboard the address or show a QR code to scan for easier usage. Avoid typing any Ethereum address manually. This is unsafe, and can easily lead to clerical errors and lost funds.
+
+1. Open your wallet app.
+2. Click on "Receive" or similarly worded option on the dashboard
+3. Copy your Ethereum address to clipboard.
+4. Provide the sender with your receiving Ethereum address.
+
+This is a general pattern, and different apps may vary or use different language. It should also be similar process if you are trying to send your funds from an exchange. 
+
+## Send cryptocurrency
+
+Would you like to send ETH to another wallet? 
+
+1. Prepare the receiving address and the network of the recipient.
+2. Open your wallet app.
+3. Click on a “Send” button on the dashboard of your wallet or similarly worded alternative.  
+4. Enter the receiving address or scan a QR code with your camera so that you don’t have to write the address manually.
+<br/><br/>
+![Send field for crypto address](./send.png)
+<br/><br/>
+
+5. Many assets, like DAI or USDC, exist on multiple networks. When transferring crypto tokens, make sure that the other side is using the same network as you are, since these are not interchangeable.
+6. Confirm the transaction and have prepared a small amount of ETH to pay for a transaction fee, the exact amount of the fee changes depending on network congestion. 
+7. Once your transaction is processed, the crypto will show up in the recipient’s account. This might take anywhere from few seconds to few minutes.
```

Comment:
Minor language additions for clarity

```suggestion
7. Once your transaction is processed, the corresponding crypto amount will show up in the recipient’s account. This might take anywhere from a few seconds to a few minutes depending on [network transaction prioritisation](/developers/docs/gas/).
```